app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -20,110 +20,108 @@
  *@
 
 @import helper._
-@import uk.gov.nationalarchives.omega.editorial.models.{ EditSetRecord, User }
-@import uk.gov.nationalarchives.omega.editorial.forms.GovukHelpers._
 @import uk.gov.hmrc.govukfrontend.views.html.components._
-@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.nationalarchives.omega.editorial.models.{EditSetRecord, User}
 
 @this(govukButton: GovukButton, govukInput: GovukInput, govukTextarea: GovukTextarea, govukFieldset: GovukFieldset, govukErrorSummary: GovukErrorSummary)
 
 @(user: User, title: String, heading: String, editSetRecordForm: Form[EditSetRecord])(implicit messages: Messages, request: Request[AnyContent])
 
 @backLink = {
-  <a class="govuk-back-link" href="/edit-set/1">Back to edit set (COAL 80 Sample)</a>
+    <a class="govuk-back-link" href="/edit-set/1">Back to edit set (COAL 80 Sample)</a>
+}
+
+@actionButtonGroup = {
+    <div class="govuk-button-group">
+        @govukButton(Button(
+            element = Some("button"),
+            name = Some("action"),
+            value = Some("save"),
+            content = Text(messages("edit-set.record.save.button"))))
+        @govukButton(Button(
+            element = Some("button"),
+            name = Some("action"),
+            value = Some("discard"),
+            classes = "govuk-button--secondary",
+            content = Text(messages("edit-set.record.discard.button"))))
+    </div>
 }
 
 @fieldsetContent = {
-  @CSRF.formField
-  @govukTextarea(Textarea(
+@CSRF.formField
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+        <h3 class="govuk-fieldset__heading">@messages("edit-set.record.edit.legend")</h3>
+    </legend>
```

Comment:
I would prefer that we use the view models as far as possible, in this case the [Legend view model](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/fieldset/Legend.scala). This is how it was done previously.

Reply:
OK. I'll try that.

Reply:
Updated.